src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -248,6 +248,7 @@ Referenced assemblies:
 - 'Microsoft.VisualStudio.ComponentModelHost, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
 - 'Microsoft.VisualStudio.CoreUtility, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
 - 'Microsoft.VisualStudio.ImageCatalog, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
+- 'Microsoft.VisualStudio.Imaging, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
```

Comment:
I wonder whether we should look at generating a baseline report for the VS2015 build too, now that our assembly referencing story is getting more complicated.
When you were getting the conditional VSShell stuff to work, didn't you hit problems where the VS2015 version built, but failed at runtime because an assembly from a newer version of VS was referenced? If so, a report might help spot that kind of error. Alternatively, we could write unit test that uses reflection to check referenced assembly versions don't exceed the target VS version (which might be simpler and more reliable).